packages/grafana-ui/src/components/index.ts

```diff
@@ -217,7 +221,7 @@ export { RelativeTimeRangePicker } from './DateTimePickers/RelativeTimeRangePick
 export { Card, Props as CardProps, getCardStyles } from './Card/Card';
 export { CardContainer, CardContainerProps } from './Card/CardContainer';
 export { FormattedValueDisplay } from './FormattedValueDisplay/FormattedValueDisplay';
-
+export { focusCss } from '../themes/mixins';
```

Comment:
Is this needed? I believe all mixins are exported by `styleMixins`?

Reply:
Didn't know that, in that case these are exports are redundant.

Reply:
Fixed, thanks for the heads up!